search-enterprise/latest/using/unsupported.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Unsupported features
+---
+
+The following features, which were supported with Search and Insight Engine 2.x and Search Services 2.x (Solr) are not supported in the latest release for Search Enterprise 3.x (Elasticsearch).
+
+## Index and re-index
+
+* Re-index of permissions and content associated with indexed nodes.
+* Indexing of nodes created during content repository bootstrap. For example, the sample site data. @Engineering what is this example?
```

Comment:
Need to check this with engineering

Reply:
@tpage-alfresco

Reply:
Changes captured from Angel's review

Reply:
This has been added based on work done with Angel